server/NucleiDetection/NucleiDetection.xml

```diff
@@ -110,4 +121,46 @@
       <default>80</default>
     </double>
   </parameters>
+  <parameters>
```

Comment:
The parameter list is getting fairly long.  I would make this (and perhaps others) `<parameters advanced="true">` to hide it by default.